effect_prn/admin/end_of_study_admin.py

```diff
@@ -53,7 +71,16 @@ class EndOfStudyAdmin(ModelAdminSubjectDashboardMixin, SimpleHistoryAdmin):
 
     list_filter = ("offschedule_datetime",)
 
-    radio_fields = {"offschedule_reason": admin.VERTICAL}
+    radio_fields = {
+        "offschedule_reason": admin.VERTICAL,
+        "cm_admitted": admin.VERTICAL,
+        "transferred_consent": admin.VERTICAL,
+        "medication_study_termination": admin.VERTICAL,
+        "dx_since_enrolment": admin.VERTICAL,
+        "dx_since_enrolment_dx": admin.VERTICAL,
+        "fcon_dose_14": admin.VERTICAL,
+        "fcyz_consolidation_phase": admin.VERTICAL,
```

Comment:
Minor point, but where order isn't important (e.g. for `radio_fields` dict), it's easier to work with if they're sorted alphabetically